hls4ml/utils/attribute_descriptions.py

```diff
@@ -0,0 +1,46 @@
+"""Strings holding attribute descriptions."""
+
+# Common attributes
+
+reuse_factor = (
+    'The number of times each multiplier is used by controlling the amount of pipelining/unrolling. '
+    'Lower number results in more parallelism and lower latency at the expense of the resources used.'
+)
+
+index = 'Internal node counter used for bookkeeping and variable/tensor naming.'
+trace = 'Enables saving of layer output (tracing).'
```

Comment:
Maybe worth saying that this only applies for hls4ml_model.predict() (to avoid any confusion with potential saving of layer outputs during simulation)